.ci/tav.json

```diff
@@ -1,39 +1,38 @@
 {
-  "// todo": "We want versions=['20','19','18','16','14','12','10','8'], but versions*modules needs to be <256 for the GH Actions jobs limit",
-  "versions": [ "20", "18", "16", "14", "12", "10", "8" ],
+  "versions": [ 20, 19, 18, 16, 14, 12, 10, 8 ],
   "modules": [
```

Comment:
Might be nice to have a `"// modules": "..."` comment that explains minVersion and suggests it should be kept in sync with .tav.yml.

Reply:
Done. We may also want this in the TAV linting 🤔